packages/react-app/src/components/pages/Bounties/Bounty/index.tsx

```diff
@@ -223,26 +220,43 @@ const BountyModal = ({
   onClose: () => void;
 }): JSX.Element => {
 	// TODO Need a Delete action if Draft or Open
-
+	const { colorMode } = useColorMode();
 	return (
-		<Modal isOpen={isOpen} onClose={onClose}>
-			<ModalOverlay />
-			<ModalContent maxW={'700px'} borderWidth={3}>
-				<ModalHeader
-					bg={useColorModeValue('gray.200', 'gray.600')}
-					roundedTop="md"
-				>
-					<BountyHeader bounty={bounty} />
-				</ModalHeader>
-				<ModalCloseButton />
-				<ModalBody>
-					<BountyDetails bounty={bounty} />
-				</ModalBody>
-				<ModalFooter>
-					<BountyActions bounty={bounty} onCancel={onClose} />
-				</ModalFooter>
-			</ModalContent>
-		</Modal>
+		
+		<Center>
+			<Modal isOpen={isOpen} onClose={onClose}>
+				<ModalOverlay />
+				<ModalContent maxW={'700px'} borderWidth={3}>
+					<ModalHeader
+						bg={useColorModeValue('gray.200', 'gray.600')}
+						roundedTop="md"
+					>
+						<BountyHeader bounty={bounty} />
+					</ModalHeader>
+					<ModalCloseButton />
+					<ModalBody>
+						<BountyDetails bounty={bounty} />
+					</ModalBody>
+					<ModalFooter>
+						<Stack direction='row' spacing={1} alignItems='center' alignContent='center' verticalAlign={'center'}>
+							<BountyActions bounty={bounty} onCancel={onClose} />
+							 <Button boxShadow={'md'}
```

Comment:
Can you add this button to the BountyActions component? That's where the rest of the buttons are. For now filter it for only DRAFT and OPEN statuses, and tooltip should say "not implemented" for now.

Also, it is in a continual loading status: 
![image](https://user-images.githubusercontent.com/2557404/184455643-299cc6d8-bc29-4ffb-9d16-ea4d818115c0.png)